lib/util/index.ts

```diff
@@ -87,7 +86,7 @@ export const mergePineOptions = <R extends {}>(
 		return defaults;
 	}
 
-	const result = cloneDeep(defaults);
+	const result = { ...defaults };
```

Comment:
We clone the top level object as we mutate it heavily, for the lower levels only $expand actually gets mutated and we handle that separately